src/lib/import-export/import/importers/importer.ts

```diff
@@ -150,8 +150,18 @@ abstract class BaseBackupImporter extends BaseImporter {
 				/^database(\/|\\)?.*/, // Match database dir and all contents
 				/^db\.php$/, // Exact match for db.php
 				/^index\.php$/, // Exact match for index.php
+				/^languages(\/|\\)?.*/, // Match languages dir and all contents
 			];
 
+			// If these directories aren't in the backup, they'll be deleted. Let's not do that.
```

Comment:
I find this comment a bit confusing.  I would suggest something like:

```suggestion
			// Directories to preserve if they are not included in the backup.
```